baker/SiteBaker.tsx

```diff
@@ -159,18 +159,6 @@ function getProgressBarTotal(bakeSteps: BakeStepConfig): number {
     let total = minimum + bakeSteps.size
     // Redirects has two progress bar ticks
     if (bakeSteps.has("redirects")) total++
-    // Add a tick for the validation step that occurs when these two steps run
-    if (bakeSteps.has("dods") && bakeSteps.has("charts")) total++
-    // Add ticks for prefetching attachments, which will only run if any of these steps are enabled
-    if (
-        bakeSteps.has("gdocPosts") ||
-        bakeSteps.has("gdocTombstones") ||
-        bakeSteps.has("dataInsights") ||
-        bakeSteps.has("authors") ||
-        bakeSteps.has("multiDimPages")
-    ) {
-        total += 9
-    }
```

Comment:
This doesn't work well when we log the line at the start of the step instead of at the end of the step, because the prefetching group steps appear in the middle of a group that first needs prefetching, which is confusing. `console.log` should be good enough for prefetching instead.